tools/pg_describe

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env node
+
+const async = require('async');
+const ERR = require('async-stacktrace');
+const colors = require('colors');
+const fs = require('fs-extra');
+const _ = require('lodash');
+const path = require('path');
+
+const databaseDescribe = require('../tests/databaseDescribe');
+
+const yargs = require('yargs')
+    .usage('Usage: $0 <database name> [options]')
+    .alias('o', 'output')
+    .nargs('o', 1)
+    .describe('o', 'Specify a directory to output files to')
+    .describe('ignore-tables', 'a list of tables to ignore')
+    .array('ignore-tables')
+    .describe('ignore-enums', 'a list of enums to ignore')
+    .array('ignore-enums')
+    .describe('ignore-columns', 'a list of columns to ignore, formatted like [table].[column]')
+    .array('ignore-columns')
+    .help('h')
+    .alias('h', 'help')
+    .example('$0 postgres')
+    .example('$0 userdb -o db_description --ignore-tables a b --ignore-columns a.col1 a.col2');
```

Comment:
For some reason on my terminal these example are wrapped to a very short line length, which makes them look horrible. This might be because there is an optional second argument to `example(command, description)` and it's leaving room for the nonexistent description. In any case I couldn't see how to fix it and it's not worth spending time on.